apollo-gradle-plugin-incubating/src/test/kotlin/com/apollographql/apollo/gradle/test/UpToDateTests.kt

```diff
@@ -72,6 +72,28 @@ class UpToDateTests {
     File(dir, "build.gradle").writeText(text.replace(apolloBlock, ""))
   }
 
+  //TODO: finish this!
```

Comment:
is this still WIP and  this TODO will be addressed later in this PR?

Reply:
Not a WIP anymore, just a sloppy commit. Thanks.